dist/leaflet-fullHash.js

```diff
@@ -0,0 +1,200 @@
+(function(window) {
```

Comment:
Wow, is this built upon the existing hash library, or does it replace it? Maybe we should call it `LayerHash`? Are we recommending that people use this to /replace/ the Hash library we've used, i guess? And are we sort of "promising" to keep it compatible with libraries that use the Hash lib? 

What do you think about releasing this as a stand-alone library, named similarly to the existing lib but like `layerHash` instead?

Just brainstorming the best way to ensure this is maintainable and compatible! Tremendous work!!!